Modern/utilities/src/main/java/org/bonej/utilities/RoiManagerUtil.java

```diff
@@ -104,15 +103,11 @@ public static boolean isActiveOnAllSlices(final int sliceNumber) {
 	// endregion
 
 	// region -- Helper methods --
-
-	private static int clamp(final int value, final int min, final int max) {
-		if (Integer.compare(value, min) < 0) {
-			return min;
-		}
-		if (Integer.compare(value, max) > 0) {
-			return max;
+	private static int clamp(final int value, final int max) {
+		if (value < 0) {
+			return 0;
 		}
-		return value;
+		return Math.min(value, max);
```

Comment:
I find it easier to read if all the return cases are spelled out:

```
if (value > max)
{
    return max;
}
return value;
```

Reply:
See above